substrate-node-example/node/Cargo.toml

```diff
@@ -27,168 +27,40 @@ jsonrpc-core = "15.1.0"
 # local dependencies
 node-template-runtime = { path = '../runtime', version = '3.0.0' }
 
-[dependencies.frame-benchmarking]
-git = 'https://github.com/paritytech/substrate.git'
-branch = 'master'
-version = '3.0.0'
```

Comment:
How is this 3.0.0 going to be declared in the new syntax? Isn't it relevant?

Reply:
The version field won't be necessary since we depend directly on the git repo